string command
string item
---
bool done
str[] reachable_urgent
str[] reachable_corridors
